tests/tracer/test_correlation_log_context.py

```diff
@@ -71,6 +72,20 @@ def test_get_log_correlation_context(self, global_config):
             "version": "test-version",
         }
 
+        tracer.context_provider.activate(
+            Context(
+                span_id=234,
+                trace_id=4321,
+            )
+        )
```

Comment:
Do we need to call `get_log_correlation_context` here?

Reply:
Good catch! 🤦